data/nord/Preset.toml

```
@@ -0,0 +1,45 @@
+slug = "nord"
+name = "Nord Theme"
+creator ="leark"
```

Minor linting nitpick:
```suggestion
creator = "leark"
```